jobs/blobstore/templates/restore.erb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -ex
+
+BOSH_DATA_DIRECTORY=${BOSH_DATA_DIRECTORY:-/var/vcap/store}
+BACKUP_DIRECTORY="$BBR_ARTIFACT_DIRECTORY"/shared
+BLOBS_DIRECTORY="$BOSH_DATA_DIRECTORY"/shared
+
+BLOBSTORE_OWNER=${BLOBSTORE_OWNER:-vcap}
+BLOBSTORE_OWNER_GROUP=${BLOBSTORE_OWNER_GROUP:-vcap}
+
+<% if p('release_level_backup') %>
+  <% [
+      link("directories_to_backup").p("cc.droplets.droplet_directory_key"),
+      link("directories_to_backup").p("cc.packages.app_package_directory_key"),
+      link("directories_to_backup").p("cc.buildpacks.buildpack_directory_key")
+    ].each do |directory_key| %>
+    rm --recursive --force ${BLOBS_DIRECTORY}/<%= directory_key %>
```

Comment:
We assume this is intentional even if ` $BACKUP_DIRECTORY/<%= directory_key %>` does not exist -- you still want to remove artifacts that have been created since the last backup point.